### PR TITLE
Reg 11/simple account page

### PR DESCRIPTION
**Pull request by Halliax**
Jira: https://olinjs.atlassian.net/browse/REG-11

### CODE REVIEW

**Comment by Halliax** [comment]
https://olinjs.atlassian.net/browse/REG-10

File: app.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
 
 // setting up routes
-const index = require('./routes/index');
+// const index = require('./routes/index');
```
Comment (billmwong) [inline app.js:8]: Hmm we'll still need this for api routes, but the linter won't like that we're including it and not using it...  I guess this is good for now :+1:

File: src/index.jsx [hunk 2]
```diff
@@ -3,14 +3,17 @@ import { render } from 'react-dom';
 import { createStore } from 'redux';
 import { Provider } from 'react-redux';
 // import App from './components/App';
-import StudentHome from './components/StudentHome';
-import reducer from './reducers/sampleReducer';
+import App from './components/App';
+import reducer from './reducers/AccountPageReducer';
```
Comment (billmwong) [inline src/index.jsx:7]: We're going to need to combine reducers when we have multiple files of reducers, but for now this is good :+1:

File: src/containers/AccountPageContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  </ul>
+);
+
+AccountPage.propTypes = {
+  settings: PropTypes.arrayOf(PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    checked: PropTypes.bool.isRequired,
+  }).isRequired).isRequired,
+  onSettingToggle: PropTypes.func.isRequired,
+};
```
Comment (billmwong) [inline src/containers/AccountPageContainer.jsx]: Put the things above this line in a separate `AccountPage.jsx` file under the components directory, this is not part of the container.

**Review by billmwong — CHANGES_REQUESTED** [review]
Mostly looks good from me!

File: src/containers/AccountPageContainer.jsx (hunk 3 above)
Comment (Halliax) [reply src/containers/AccountPageContainer.jsx]: Can do. I think I was working off the example code, which was probably just there to show how they link together

Comment (Halliax) [reply src/containers/AccountPageContainer.jsx]: fixed

File: src/containers/AccountPageContainer.jsx [hunk 4]
```diff
@@ -1,28 +1,6 @@
-import React, { PropTypes } from 'react';
```
Comment (billmwong) [inline src/containers/AccountPageContainer.jsx]: I think you still need to import React in here, but I could be wrong. Have you tested this?

File: app.js (hunk 1 above)
Comment (Halliax) [reply app.js:8]: Yeah I'd like to rewrite that routing later, it's pretty gross. For now it's what worked with react-router

File: src/containers/AccountPageContainer.jsx (hunk 4 above)
Comment (Halliax) [reply src/containers/AccountPageContainer.jsx]: I did test it, and the linter complained when i left React in

Comment (Halliax) [reply src/containers/AccountPageContainer.jsx]: Maybe it's the react-redux bindings that make it still work?

Comment (billmwong) [reply src/containers/AccountPageContainer.jsx]: I'd buy that, but if it works then cool

**Review by billmwong — APPROVED** [review]
Waiting on a student review

**Review by arianaolson419 — APPROVED** [review]
It all looks good to me! Thank you for getting react router started, I wasn't sure when we were incorporating that.

**Pull request merged**